app/pages/unsubscribe.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { Component, PropTypes } from 'react';
+import Auth from 'panoptes-client/lib/auth';
+
+
+class UnsubscribeFromEmails extends Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.renderForm = this.renderForm.bind(this);
+    this.renderIconFeedback = this.renderIconFeedback.bind(this);
+    this.renderTextFeedback = this.renderTextFeedback.bind(this);
+    this.renderTokenFlowFeedback = this.renderTokenFlowFeedback.bind(this);
```

Comment:
You don't have to bind the render methods here: they're already bound to the instance. Only the event handlers need to be bound.